README.md

```diff
@@ -30,24 +30,22 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) assists with verifying Free and Open Source Software license compliance by checking a
-project's source code and its dependencies.
-
-From a bird's eye, it works by analyzing the project's build system for dependencies, downloading the source code of the
-dependencies, scanning all source code for license information, and summarizing the results.
-
-The different tools that make up ORT are designed as libraries (for programmatic use), with a minimal command line
-interface (for scripted use).
-
-The toolkit consists of the following tools:
-
-* [_Analyzer_](#analyzer) - determines dependencies of a project. Supports multiple package managers and sub-projects.
-  No changes to the projects are required.
-* [_Downloader_](#downloader) - fetches the source code referred to by the Analyzer result.
-* [_Scanner_](#scanner) - wraps existing license / copyright scanners to detect findings in local source code
-  directories.
-* [_Evaluator_](#evaluator) - evaluates license findings against customizable policy rules.
-* [_Reporter_](#reporter) - presents results in various formats such as visual reports, open source notices or
+The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
+compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+
+It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
+These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
```

Comment:
I'd suggest to either move this to the line above or add another empty line to reflect how the text is rendered.
Shall we say "Kotlin JVM libraries"?

Reply:
Hmm, what's wrong with the rendering? For me these sentences are rendering in one line like "[...] services. These [...]" as it should be.

Regarding "Kotlin JVM libraries", we already say in the "Development" section that Kotlin is used, which is a more fitting place IMO. Also I do not want really get as detailed as "JVM" here, but just highlight the two "user facing" interfaces, libraries and CLI.

Reply:
Sorry, I accidentally edited your comment instead of adding a new one and it seems there is no way to revert that.

Reply:
>Hmm, what's wrong with the rendering? For me these sentences are rendering in one line like "[...] services. These [...]" as it should be.

My point was to format them here the same way they are rendered, but it's not that important. I just find it clearer.

Reply:
Ah, ok, but didn't we agree to hard-wrap also Markdown at column 120?